layers/+source-control/version-control/packages.el

```diff
@@ -54,7 +54,8 @@
       ;; If you enable global minor mode
       (when (and (eq version-control-diff-tool 'git-gutter)
                  version-control-global-margin)
-        (global-git-gutter-mode t))
+        (run-with-idle-timer 5 nil (lambda ()
```

Comment:
Actually, this will cause git-gutter and git-gutter+ to load for everyone. I'd need to set the defer conditionally and I'm not sure of the impact of that if one were byte compiled that file (since use-package expands at compile time)